cola/libdialect/constraints.cpp

```diff
@@ -72,6 +72,9 @@ SepDir dialect::negateSepDir(SepDir sd) {
         return SepDir::RIGHT;
     case SepDir::UP:
         return SepDir::DOWN;
+    default :
+        COLA_ASSERT(false);
+        return SepDir::WEST;
```

Comment:
Do we really need a `return`? It looks like at line 101 we get away without one.